js/src/secureApi.js

```diff
@@ -40,6 +42,10 @@ export default class SecureApi extends Api {
     this._tryNextToken();
   }
 
+  get transport () {
```

Comment:
Is there any reason we are re-defining this here? Api has both `this._transport` as well as `get transport` already and SecureApi only extends Api.